layouts/partials/main/pages.html

```diff
@@ -0,0 +1,21 @@
+{{ with .Page.Pages }}
```

Comment:
This is the new template partial that adds a table of contents list to pages like https://docs.blubracket.com/how-to/ and https://docs.blubracket.com/faq/

It's called from `layouts/_default/list.html`